src/components/ColorPicker.css

```diff
@@ -81,33 +81,68 @@
   display: flex;
   align-items: center;
   justify-content: center;
+  z-index: 1;
+  position: relative;
+}
+.color-input-container:focus-within .color-picker-hash {
+  box-shadow: 0 0 0 2px #a5d8ff;
+}
+.color-input-container:focus-within .color-picker-hash::before,
+.color-input-container:focus-within .color-picker-hash::after {
+  content: "";
+  width: 1px;
+  height: 100%;
+  position: absolute;
+  top: 0;
+}
+.color-input-container:focus-within .color-picker-hash::before {
+  background: #dee2e6;
+  right: -1px;
+}
+.color-input-container:focus-within .color-picker-hash::after {
+  background: white;
+  right: -2px;
 }
 
 .color-input-container {
   display: flex;
 }
 
 .color-picker-input {
-  width: 6.25em;
+  width: 12ch; /* length of `transparent` + 1 */
+  margin: 0;
   font-size: 1rem;
   color: #343a40;
   border: 0px;
   outline: none;
   height: 1.75em;
   box-shadow: #dee2e6 0px 0px 0px 1px inset;
-  box-sizing: content-box;
   border-radius: 0px 4px 4px 0px;
   float: left;
   padding: 1px;
   padding-left: 0.5em;
   appearance: none;
+  font-family: Cascadia, monospace;
```

Comment:
At one point I also made that change, but @vjeux convinced me (and I agree) that we should not use Cascadia for "consistency" here..